.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+name: test
+on: [push, pull_request]
+env:
+  SERVER_AUTH_TYPE: basic
+  SERVER_URL: http://127.0.0.1:5984
+  SERVER_USERNAME: admin
+  SERVER_PASSWORD: password
+jobs:
+  test:
+    strategy:
+      matrix:
+        java: ['8'] 
+        # java: ['8', '11', '17'] 
```

Comment:
I assume you are planning to re-enable the full matrix when the review is complete?

Reply:
yes